Makefile

```diff
@@ -120,9 +122,20 @@ install: manifests
 uninstall: manifests
 	kustomize build config/crd | kubectl $(KUBECTL_ARGS) delete -f -
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: install manifests
+# Apply config to the local development environment based on environment
+# variables.
+config/development/kustomization.yaml: config/development/kustomization.yaml.sample
+	cp config/development/kustomization.yaml.sample config/development/kustomization.yaml
 	cd config/development && kustomize edit set image foundationdb/fdb-kubernetes-operator=${IMG}
+ifneq "$(IMG_PREFIX)" ""
```

Comment:
I don't see where the variable `IMG_PREFIX` is. defined in the makefile. It's also arguably confusing how `IMG` and `IMG_PREFIX` work together (since it's not documented and the operator image doesn't make use of it).